README.md

```diff
@@ -267,19 +267,33 @@ Create an instance of the engine and transcribe an audio file:
 #include "pv_leopard.h"
 
 pv_leopard_t *handle = NULL;
-pv_status_t status = pv_leopard_init("${ACCESS_KEY}", "${MODEL_PATH}", &handle);
+bool automatic_punctuation = false;
```

Comment:
I believe they need stdbool to use a bool, so we should add this to the includes